### PR TITLE
Test/new user in database/pf 100

### PR DESCRIPTION
**Pull request by BalazsiOliver**

### CODE REVIEW

**Comment by bsnAndras** [comment]
https://github.com/gfa-cc-after/StudyGroup_1_PET_Clinic_Backend-main/blob/3225320050aaf884c49017d32874bb85a2a8f465/src/main/java/com/greenfox/dramacsoport/petclinicbackend/services/AppUserServiceImpl.java#L94C4-L99C10

Everything is working nicely.
Maybe we should also check for he unhappy paths of the ```isPasswordLongerThanThreeChar()``` and the ```isUserRegistered() ``` methods.

**Review by bsnAndras — COMMENTED** [review]
https://github.com/gfa-cc-after/StudyGroup_1_PET_Clinic_Backend-main/blob/3225320050aaf884c49017d32874bb85a2a8f465/src/main/java/com/greenfox/dramacsoport/petclinicbackend/services/AppUserServiceImpl.java#L94C4-L99C10

Everything is working nicely.
Maybe we should also check for he unhappy paths of the isPasswordLongerThanThreeChar() and the isUserRegistered()  methods.

I will throw in another thing for discussion: Should we test for the controller part too, or should it be another ticket?

**Comment by BalazsiOliver** [comment]
I think we dont need another ticket for that, i will look forward to do it aswell.

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+SPRING_MAIL_PASSWORD=eoma lkuz tjte bmku
```
Comment (markkovari) [inline .env:1]: image

**Review by markkovari — APPROVED** [review]

**Review by bsnAndras — APPROVED** [review]
I removed some unused imports and such, if you don't mind.
Otherwise perfect! Good to go!

**Comment by BalazsiOliver** [comment]
Alltough the test are not perfectin my opinion, i think they work properly and do their jobs well. If you think its enough we are ready to merge this branch.

**Pull request merged**